connectors/google-sheets/element-templates/google-sheets-connector.json

```diff
@@ -0,0 +1,586 @@
+          "value": "ROWS"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.dimension"
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addEmptyColumnOrRow"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Row index",
+      "description": "Enter row index",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must be a number."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "getRowByIndex"
+      }
+    },
+    {
+      "label": "Row index",
+      "description": "Enter the index of the row to add",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must contains only numbers."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "createRow"
+      }
+    },
+    {
+      "label": "Column index",
+      "description": "Enter the index of the column",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100,
+        "pattern": {
+          "value": "[A-Z]+",
+          "message": "Must contains only capital letters."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "deleteColumn"
+      }
+    },
+    {
+      "label": "Start index",
+      "description": "Enter start index (leave empty if add to the end of the sheet)",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.startIndex"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must be a number."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addEmptyColumnOrRow"
+      }
+    },
+    {
+      "label": "End index",
+      "description": "Enter End index (leave empty if add to the end of the sheet)",
```

Comment:
can you add here an example

Reply:
a short example is always helpfull and a faster solution for our enduser. A good example is 
![image](https://github.com/camunda/connectors-bundle/assets/119578341/e804791e-1b93-44c6-9c12-bfbeea2469c4)

here we are saying, what we are expecting and referencing the documentation for follow up questions

Reply:
Add link to the documentation where it'll be described
<img width="376" alt="image" src="https://github.com/camunda/connectors-bundle/assets/131355515/788e3b9e-13c2-4234-8047-3ddbf41554c9">